apps/portlet-documents/src/main/webapp/vue-app/attachment/extensions.js

```diff
@@ -40,6 +41,13 @@ export function installExtensions() {
     rank: 10,
   });
 
+  extensionRegistry.registerExtension('CommonCkeditor', 'ckeditor-extensions', {
+    id: 'selectImage',
+    extraPlugin: 'selectImage',
+    extraToolbarItem: 'selectImage',
+    rank: 20,
+  });
```

Comment:
```suggestion
  extensionRegistry.registerExtension('WYSIWYGPlugins', 'image', {
    id: 'selectImage',
    extraPlugin: 'selectImage',
    extraToolbarItem: 'selectImage',
    rank: 20,
  });
```

IMO, this extension name isn't adequate, you should add a specific extension name to image like `CkeditorPlugins` /  `image`